staging/src/k8s.io/mount-utils/mount_helper_unix.go

```diff
@@ -32,7 +32,10 @@ const (
 	// At least number of fields per line in /proc/<pid>/mountinfo.
 	expectedAtLeastNumFieldsPerMountInfo = 10
 	// How many times to retry for a consistent read of /proc/mounts.
-	maxListTries = 3
+	// Failed consistent read is not retried in case of volume reconstruction after kubelet restart,
+	// which may potentially leave volumes mounted and abandoned by kubelet.
+	// We should try really hard not to fail here.
+	maxListTries = 999
```

Comment:
how long it will roughly take for this max tries?

Reply:
It's around 1-2s. But my intent is infinity, any error on reconstructing a volume may leave a volume mounted and force detached.

Reply:
It is a very tight loop without  any sort of exp. backoff (and adding exp. backoff is not feasible anyways here) and hence could result in high cpu usage while this loop runs. That is kind of my main concern here.

Reply:
This also reminds me of https://github.com/kubernetes/kubernetes/pull/61776 implementation of reading `/proc/mounts`. cc @cofyc

Reply:
I checked #61776, it could actually speed up volume reconstruction (all the volumes need to read `/proc/mounts` after kubelet start and caching could help), but it won't help us to get a consistent content of the file. Reading `/proc/mount` is so fast that IMO the difference with caching would be negligible.